### DISCUSSION TITLE
Add a option to pick up cards based on print number

### DISCUSSION THREAD

**Discussion by Vertigo1207** [NoMeansNowastaken/KarutaSniper#13] [Polls]
please

**Comment by splat58** [comment]
Ill add it if I have time later today, should be easy enough

**Reply by Vertigo1207** [reply]
lets go, thank you so much man

**Reply by NoMeansNowastaken** [reply]
i see there are two seperate print numbers so what do you want to target? first one? last one? both? (I have no idea what they do)

**Reply by Vertigo1207** [reply]
target the one that has alot of numbers

**Reply by Vertigo1207** [reply]
what you are aiming for is if that number is under 1000, it should try and claim it no matter the card

**Reply by NoMeansNowastaken** [reply]
ill allow the number to be configurable, but default to 1k

**Reply by NoMeansNowastaken** [reply]
so ive added this to the beta branch instead because its not finished and doesn't work all the time or gives errors. Please note that the beta branch doesn't work atm however.

**Reply by Vertigo1207** [reply]
sweet as
